### PR TITLE
Fix for mark plan as complete error

### LINKED ISSUE #616: Mark Plan as complete errors out
issue first raised [here](https://smartregister.atlassian.net/browse/RVL-699)

Clicking confirm on the plan's mark as complete initiates a PUT fetch api call to `rest/plans` that errors out with `500` status.

### PR DESCRIPTION
**Pull request by p-netm**
closes #616 

Refactors type of received response during 'get' request to `'rest/plans/<identifier>`, Fixes the payload to `PUT` accordingly.

### CODE REVIEW

File: src/containers/forms/PractitionerForm/UserIdSelect/index.tsx [hunk 1]
```diff
@@ -102,10 +102,18 @@ export const UserIdSelect: React.FC<Props> = props => {
...
+      .map((user: OpenMRSUser) => ({
+        label: user.display,
+        value: user.uuid,
+      }))
+      .sort((userA, userB) => {
+        const userALabel = userA.label;
+        const userBLabel = userB.label;
+        return userALabel === userBLabel ? 0 : userALabel > userBLabel ? 1 : -1;
+      });
+  }, [openMRSUsers]);
```
Comment (moshthepitt) [inline src/containers/forms/PractitionerForm/UserIdSelect/index.tsx]: Is there a test that covers this?

File: src/containers/pages/PractitionerViews/CreateEditPractitioner/index.tsx [hunk 2]
```diff
@@ -65,7 +66,7 @@ const CreateEditPractitionerView = (props: PropsTypes) => {
   };
   const breadcrumbProps: BreadCrumbProps = {
     currentPage: {
-      label: editing ? `${EDIT} ${PRACTITIONER}` : `${NEW} ${PRACTITIONER}`,
+      label: editing ? `${EDIT} ${PRACTITIONER}` : `${capitalize(NEW as string)} ${PRACTITIONER}`,
```
Comment (moshthepitt) [inline src/containers/pages/PractitionerViews/CreateEditPractitioner/index.tsx]: why not use `toUpperCase()`?

**Comment by p-netm** [comment]
Closed unintentionally, created another pr #618. this cannot be reopened since i overwrote the branch

File: src/containers/forms/PractitionerForm/UserIdSelect/index.tsx (hunk 1 above)
Comment (p-netm) [reply src/containers/forms/PractitionerForm/UserIdSelect/index.tsx]: Added one.

Comment (moshthepitt) [reply src/containers/forms/PractitionerForm/UserIdSelect/index.tsx]: @p-netm  where? (I am assuming we are not using this branch for anything)

**Pull request closed without merging**